static/app/components/events/eventTags/eventTagsTree.tsx

```diff
@@ -231,13 +231,9 @@ export const TreeColumn = styled('div')`
   display: grid;
   grid-template-columns: minmax(auto, 175px) 1fr;
   grid-column-gap: ${space(3)};
-  &:first-child {
-    margin-left: -${space(1)};
-  }
```

Comment:
This might need to be retained, these negative margins are so that the header text and the tag text (with alternating white/gray backgrounds) are aligned, rather than having the header text aligned with the background itself

like in this 'after' photo from your pr text, the text `url` is indented compared to `Highlights` which might not be intended (though if it is, feel free to dismiss)

<img width="1218" height="706" alt="Image" src="https://github.com/user-attachments/assets/69da8948-88cd-4c68-8def-caf442ab1f5b" />

Reply:
yeah, just took a look at the figma doc, it'll need the negative margin (or something similar) to align
<img width="436" height="238" alt="image" src="https://github.com/user-attachments/assets/0f265280-f648-4063-809c-9e9a93907c92" />